### PR TITLE
Comment-form component

### LINKED ISSUE #78: Add Comment Form Component
User Stories:

- #5

### PR DESCRIPTION
**Pull request by Sagi-Viktor**
closes #78

### CODE REVIEW

File: angular/src/app/components/comment-form/comment-form.component.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  margin-left: 85%;
+}
+
+.comment-head {
+  vertical-align: top;
+}
+
+.comment-head-text {
+  display: inline;
+}
```
Comment (benedekhalaj) [inline angular/src/app/components/comment-form/comment-form.component.css:36]: Lot of styles, great job!

File: angular/src/app/components/comment-form/comment-form.component.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.comment-form {
```
Comment (dudaskobendeguz) [inline angular/src/app/components/comment-form/comment-form.component.css:1]: nice CSS <3

File: angular/src/app/components/comment-form/comment-form.component.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <img class="profile-picture" alt="{{user.username}}'s profile picture" src="{{user.imageSource}}">
+    <div class="comment-head-text">
+      <p class="user-full-name">{{user.firstName}} {{user.lastName}}</p>
+      <p class="username">@{{user.username}}</p>
+    </div>
+  </div>
+  <div class="separator"></div>
+
+  <form [formGroup]="checkoutForm" (ngSubmit)="addComment()">
+    <textarea id="description" class="description" formControlName="description"></textarea>
```
Comment (benedekhalaj) [inline angular/src/app/components/comment-form/comment-form.component.html:12]: I like the textarea solution, great UX

**Review by benedekhalaj — APPROVED** [review]
Great job!

**Review by dudaskobendeguz — APPROVED** [review]

**Pull request merged**